src/pybind/mgr/rook/module.py

```diff
@@ -413,11 +413,6 @@ def _service_add_decorate(self, typename, spec, func):
             mgr=self
         )
 
-    def add_nfs(self, spec):
-        # type: (NFSServiceSpec) -> RookCompletion
-        return self._service_add_decorate("NFS", spec,
-                                          self.rook_cluster.add_nfsgw)
-
```

Comment:
Do we want to remove these or try to fix them per #37771?

Reply:
> Do we want to try to fix these per #37771?

I have closed that PR.